_P208_Nokia_LCD_5110.ino

```diff
@@ -217,32 +224,45 @@ boolean Plugin_208(byte function, struct EventStruct *event, String& string){
             argIndex = string.lastIndexOf(',');
             tmpString = string.substring(argIndex + 1);
             if (tmpString.equalsIgnoreCase(F("Clear"))){
-              addLog(LOG_LEVEL_INFO, F("Clear Display"));
+              myLog("Clear Display");
               lcd3->clearDisplay();
               lcd3->display();
             }
             if (tmpString.equalsIgnoreCase(F("blOn"))){
               success = true;
               PCONFIG(0) = 1;
-              setBacklight(event); 
+              setBacklight(event);
             }
             if (tmpString.equalsIgnoreCase(F("blOff"))){
               success = true;
               PCONFIG(0) = 0;
-              setBacklight(event); 
+              setBacklight(event);
             }
-          break;
-        }
+            break;
+          }
         }
       }
   }  // switch (function)
   return success;
 }
 
+//void myLog (String message) {
+//  addLog(LOG_LEVEL_INFO, "P208: " + message);
+//}
+void myLog(const String & message) {
+  String log;
+  if (sentlog){
+    log.reserve(message.length() + 6);
+    log += F("P208: ");
+    log += message;
+    addLog(LOG_LEVEL_INFO, log);

```

Comment:
when compiling on current `mega` branch, please use `addLogMove` macro when appropriate

Reply:
I can not use the current ESPmega (ESPEasy-mega-20220328) because of build errors......
I'm using ESPEasy_ESP32_mega-20211224 without build errors....

Reply:
What build errors do you have?

Reply:
![unnamed](https://user-images.githubusercontent.com/79747798/161437965-25e774ac-7820-4950-85f1-557ec611fa37.png)

Reply:
I've tried to replay the scenario you have most likely taken:
- Downloaded the ESPEasy-20220328 sources zip-file from the [Releases](https://github.com/letscontrolit/ESPEasy/releases) page
- Unzipped that file using the 7-zip tool (I _always_ avoid using the Windows explorer .zip extracter, as that thing has been dangerously broken since the first ever release around Windows XP!)
- Got the `_P208_Nokia_LCD_5110.ino` from this PR
- Added `Adafruit PCD8544 Nokia 5110 LCD library` to `lib_deps` in both `platformio_esp32_envs.ini` and `platformio_esp82xx_base.ini` (don't forget to add a comma after the last entry already there)
- Added `#define USES_P208` to the `#ifdef PLUGIN_BUILD_NORMAL` section of `define_plugin_sets.h` to include the plugin (a bit hacky, I know 👼)
- Clicked Build for the `normal_ESP8266_4M1M` PIO configuration: successful on first execution
- Clicked Build for the `normal_ESP32_4M316k` PIO configuration: successful after a few retries (PIO is known to sometimes have issues when switching configuration like this)

Not sure what's the difference is with what you did, but it should be mostly the same, possibly you unpacked the sources-zip using the default Windows tool?